test/libsolidity/smtCheckerTests/simple/failing_test.sol

```diff
@@ -0,0 +1,10 @@
+contract C {
+
+    function test() public {
+        assert(false);
+    }
+
+}
+// ====
+// SMTEngine: all
```

Comment:
I think here we should force engine = BMC and solver = cvc5

Reply:
I just wanted to test which jobs actually run SMTChecker tests (looks like `t_osx_soltest`, `t_ubu_force_release_soltest_all` and `t_ubu_soltest_all`).

But your question got me thinking. If we include tests that explicitly require `cvc5`, then all the CI images where we want to run these tests (Ubuntu and OSX) must have `cvc5` available, right?